config/prod.webpack.config.js

```diff
@@ -27,6 +29,24 @@ plugins.push(
   )
 );
 
+if (process.env.SENTRY_AUTH_TOKEN) {
+  plugins.push(
+    sentryWebpackPlugin({
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+      org: process.env.SENTRY_ORG,
+      project: process.env.SENTRY_PROJECT,
+    })
+  );
+
+  const SourceMapsPlugin = new webpack.SourceMapDevToolPlugin({
+    test: /\.js/i,
+    exclude: /(node_modules|bower_components)/i,
+    filename: `sourcemaps/[name].js.map`,
+  });
+
+  plugins.push(SourceMapsPlugin);
```

Comment:
I think this plugin is not needed. Sentry Webpack plugin already generate the source maps, isn't it? I have tried to remove this plugin and check the /dist folder and could see `sourcemaps` folder created successfully.